<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateDialog</class>
 <widget class="QDialog" name="TemplateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QImageViewerWithAnchor" name="lb_selectTemplates"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <widget class="QPushButton" name="pb_setTemplates">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_anchorTemplates">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>锚定</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QImageViewerWithAnchor</class>
   <extends>QFrame</extends>
   <header>QImageViewerWithAnchor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
